client/views/flashcards/FlashcardNew.html

```diff
@@ -0,0 +1,19 @@
+<template name="FlashcardNew">
+<div class="input-group input-group-sm text-center">
+  {{#autoForm id="new-card" type="insert" collection="Flashcards"}}
```

Comment:
In the commits before this, you had a <form> tag before the {{autofrom}} helper. This created two <form> tags in the html after rendering. That's why some of the styles are a bit wonky now.